.eslintrc.js

```diff
@@ -4,25 +4,31 @@ module.exports = {
     'eslint:recommended',
     'plugin:import/recommended',
     'plugin:import/typescript',
-    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
-    'plugin:prettier/recommended', // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
```

Comment:
I think it's good to have such comments

Reply:
I've added the prettier one back in. The other one is not adding anything IMO